external-crates/move/crates/move-compiler/src/naming/ast.rs

```diff
@@ -358,7 +358,7 @@ pub type LambdaLValues = Spanned<LambdaLValues_>;
 #[derive(Debug, PartialEq, Clone)]
 pub enum ExpDotted_ {
     Exp(Box<Exp>),
-    Dot(Box<ExpDotted>, Field),
```

Comment:
Nit: `loc` should be in the middle, with a comment indicating what the location is